protocol-units/sequencing/memseq/src/lib.rs

```diff
@@ -1,241 +1,469 @@
-pub use movement_types::{Block, Transaction, Id};
 use mempool_util::{MempoolBlockOperations, MempoolTransactionOperations};
-use std::{path::PathBuf, sync::Arc};
-use tokio::sync::RwLock;
 pub use move_rocks::RocksdbMempool;
+pub use movement_types::{Block, Id, Transaction};
 pub use sequencing_util::Sequencer;
+use std::{path::PathBuf, sync::Arc};
+use tokio::sync::RwLock;
 
 #[derive(Clone)]
-pub struct Memseq<T : MempoolBlockOperations + MempoolTransactionOperations> {
-    pub mempool : Arc<RwLock<T>>,
-    // this value should not be changed after initialization
-    block_size : u32,
-    pub parent_block : Arc<RwLock<Id>>,
-    // this value should not be changed after initialization
-    building_time_ms : u64,
+pub struct Memseq<T: MempoolBlockOperations + MempoolTransactionOperations> {
+	pub mempool: Arc<RwLock<T>>,
```

Comment:
Can we settle on indentation with spaces (I think the default with rustfmt)? This does not look like it does in the editor.

Reply:
I believe it's because of the rule `hard_tabs = true` set here `https://github.com/movementlabsxyz/movement/blob/main/rustfmt.toml` and probably your editor is using the stable channel for `fmt`.

I would personally vote for using `cargo +nightly` for fmt. And then we all set our IDE's to use that. But happy to let the rest of the team weigh in on this.

Reply:
Yes, the setting `hard_tabs = true` enforces using tabs instead of spaces. Personally, I also prefer the use of spaces. For now, I will just save using the provided `rustfmt` config. As such, I will not change anything until we have a consensus on which `rustfmt` configuration we are going to use, and then we can do one big pull request to fix the formatting.

Reply:
Yep we can use spaces and get rid of that rule.